website/versioned_docs/version-27.4/ECMAScriptModules.md

```diff
@@ -12,9 +12,13 @@ Jest ships with _experimental_ support for ECMAScript Modules (ESM).
 With the warnings out of the way, this is how you activate ESM support in your tests.
 
 1. Ensure you either disable [code transforms](./configuration#transform-objectstring-pathtotransformer--pathtotransformer-object) by passing `transform: {}` or otherwise configure your transformer to emit ESM rather than the default CommonJS (CJS).
-1. Execute `node` with `--experimental-vm-modules`, e.g. `node --experimental-vm-modules node_modules/jest/bin/jest.js` or `NODE_OPTIONS=--experimental-vm-modules npx jest` etc.. On Windows, you can use [`cross-env`](https://github.com/kentcdodds/cross-env) to be able to set environment variables.
-1. Beyond that, we attempt to follow `node`'s logic for activating "ESM mode" (such as looking at `type` in `package.json` or `.mjs` files), see [their docs](https://nodejs.org/api/esm.html#esm_enabling) for details.
-1. If you want to treat other file extensions (such as `.jsx` or `.ts`) as ESM, please use the [`extensionsToTreatAsEsm` option](Configuration.md#extensionstotreatasesm-arraystring).
+1. Execute `node` with `--experimental-vm-modules`, e.g. `node --experimental-vm-modules node_modules/jest/bin/jest.js` or `NODE_OPTIONS=--experimental-vm-modules npx jest` etc.. 
+    
+    On Windows, you can use [`cross-env`](https://github.com/kentcdodds/cross-env) to be able to set environment variables.
```

Comment:
why the indentation change?

Reply:
4 spaces allow "On Windows…" text to remain a separate paragraph, but still "belong" to this bullet point.

Reply:
No additional line break, 2 or 4 spaces:
![image](https://user-images.githubusercontent.com/5426427/147740073-43567406-8862-4d7c-a964-cb902bab1989.png)

With additional line break, 2 spaces:
![image](https://user-images.githubusercontent.com/5426427/147740132-0a5bdcc2-2f52-4df8-8130-c2de1157acd9.png)

(current) With additional line break, 4 spaces:
![image](https://user-images.githubusercontent.com/5426427/147740165-5d8ee18b-e49f-4050-b38e-8300fc0efa9f.png)